src/cpp/include/openvino/genai/continuous_batching_pipeline.hpp

```diff
@@ -61,6 +61,8 @@ class OPENVINO_GENAI_EXPORTS ContinuousBatchingPipeline {
     GenerationHandle add_request(uint64_t request_id, const ov::Tensor& input_ids, const ov::genai::GenerationConfig& sampling_params);
     GenerationHandle add_request(uint64_t request_id, const std::string& prompt, const ov::genai::GenerationConfig& sampling_params);
 
+    std::string get_model_configuration_string();
```

Comment:
I'm against of adding API method for debug purposes.

Can we enable it via env var?

Reply:
Env var would be problematic. Maybe we can add a flag in the constructor that is set to false by default ?

Reply:
why is it problematic? once model is compiled, we can dump this information based on env var.

Reply:
Because we do not handle ENV variables in OVMS runtime, this is extra capability reserved only for CLOUD services for example secrets or endpoints. We can add a graph llm options parameter, then we can pass it to the constructor.

Reply:
I think we should store loaded model configuration and provide getter for it. 
In current shape it indeed looks like a debug method as the string it returns is not very useful except for printing. 
I would change it to return a map with properties represented as {key, value}. This way user would be able to conveniently check model configuration and possibly make decisions in their own applications based on value of certain properties. It might also be useful for UX and monitoring purposes.
The `full_log` flag in pipeline constructor is unnecessary in my opinion. I think we should store configuration either way.

Reply:
I would consider this method as informative so not only for debugging purposes.  The output of the method in a form of vector of strings can be used for various purposes on the client application side just like such API is in OV Runtime. 
Introducing extra env variable OV_CB_FULL_LOG to mute the output of the method is in my opinion totally useless. I suggest to keep the same output from get_model_configuration(). It will be the client application to use it or not. 
It will be used in OVMS to track the configuration of the model for monitoring and benchmarking purposes.